### PR TITLE
Upgrade to Ruby 3 and Rails 6.1

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,8 +10,9 @@ end
 gem 'deprecation'
 gem 'honeybadger'
 gem 'puma', '~> 5.3' # the app server
-gem 'rails', '~> 5.2.0'
-gem 'roo', '>= 2.7.1'
+gem 'rails', '~> 6.1.0'
+# Pin roo dependency to a thus far unreleased commit that brings Ruby 3 compatibility
+gem 'roo', github: 'roo-rb/roo', ref: '868d4ea419cf393c9d8832838d96c82e47116d2f' # gem 'roo', '>= 2.7.1'
```
Comment (ndushay) [inline Gemfile:15]: ew - are we really using roo???   why?

File: config/application.rb [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+# require "active_job/railtie"
+# require "active_record/railtie"
+# require "active_storage/engine"
 require 'action_controller/railtie'
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
+# require "action_view/railtie"
+# require "action_cable/engine"
+# require "sprockets/railtie"
```
Comment (ndushay) [inline config/application.rb:15]: I guess it's okay to keep all these comments.

**Review by ndushay — APPROVED** [review]
whatevs.

File: Gemfile (hunk 1 above)
Comment (mjgiarlo) [reply Gemfile:15]: I could prolly `git blame` it but 🤷🏻‍♂️ ... yeah

Comment (jcoyne) [reply Gemfile:15]: To parse spreadsheet uploads

**Pull request merged**